translations/de.json/de.json

```
@@ -0,0 +1,129 @@
+{
+	"Catalyseur d'énergies pour OpenStreetMap": "Energie-Spender für OpenStreetMaps",
+	"Projet du mois OpenStreetMap": "OpenStreetMap-Projekt des Monats",
+	"Copyright 2020 ProjetDuMois.fr - Développé sous licence libre AGPL 3.0": "Copyright 2020 ProjetDuMois.fr - Entwickelt unter der freien Lizenz AGPL 3.0",
+	"Un projet supporté par l'association": "Ein Projekt, das von vom Verein unterstützt wird",
+	"Ce projet adhère aux": "Dieses Projekt unterstützt die",
+	"Principes pour le développement numérique": "Grundsätze für die digitale Entwicklung",
+	"Une erreur s'est produite à la connexion, veuillez réessayer": "Bei der Verbindung ist ein Fehler aufgetreten, bitte versuchen Sie es noch einmal",
+	"Rechercher un lieu...": "Nach einem Ort suchen...",
+	"Plus de détails": "Weitere Details",
+	"Avec l'éditeur intégré": "Mit dem integrierten Editor",
+	"Avec une appli mobile": "Mit einer mobilen App",
+	"Avec iD": "Mit iD",
+	"Avec JOSM": "Mit JOSM",
+	"Via Osmose": "Via Osmosis",
+	"Via une note de carte": "Über einen Kartenhinweis",
+	"Via OSM.org": "Über OSM.org",
+	"Via Mapillary": "Mit Mapillary",
+	"Retour à la page d'accueil": "Zurück zur Startseite",
+	"Projet du mois": "Projekt des Monats",
+	"Carte": "Karte",
+	"Liste": "Liste",
+	"Afficher uniquement les signalements": "Nur Meldungen anzeigen",
+	"Prêt à intégrer": "Bereit zur Integration",
+	"Avec photo": "Mit Foto",
+	"Anonymes": "Anonymisiert",
+	"Aucun signalement pour l'instant !": "Bisher gibt es keine Meldungen.",
+	"commentaire(s)": "Kommentar(e)",
+	"Anonyme": "Anonymisiert",
+	"Ouvert": "Offen",
+	"Fermé": "Geschlossen",
+	"Réouvert": "Wiedereröffnet",
+	"Attributs renseignés ;": "Angegebene Attribute;",
+	"Compléter": "Ergänzen",
+	"Éditer": "Bearbeiten",
+	"Se déconnecter": "Abmelden",
+	"S'inscrire / Se connecter": "Registrieren / Anmelden",
+	"Utilisateur": "Benutzer",
+	"Notifications non-lues": "Nicht gelesene Mitteilungen",
+	"Notifications": "Benachrichtigungen",
+	"Mes badges": "Meine Abzeichen",
+	"Merci !": "Danke!",
+	"Podium": "Podium",
+	"Inscrivez-vous ou connectez-vous pour voir le podium": "Registrieren Sie sich oder melden Sie sich an, um das Podium zu sehen",
+	"Pseudo": "Nickname",
+	"Points": "Punkte",
+	"Avancement": "Fortschritt",
+	"Notes": "Hinweise",
+	"Clés fréquentes": "Häufige Schlüssel",
+	"jours restants": "Noch verbleibende Tage",
+	"jours de contributions": "Anzahl Tage mit Beiträgen",
+	"personnes ayant contribué": "Personen, die einen Beitrag geleistet haben",
+	"En ce moment": "Zur Zeit",
+	"Statistiques": "Statistik",
+	"Données": "Daten",
+	"Retour à l'accueil": "Zurück zur Startseite",
+	"Accueil | Découvrir le projet du mois": "Startseite | Entdecken Sie das Projekt des Monats",
+	"Erreur %s ; rien que le désert ici...": "Fehler %s; hier ist nur Wüste...",
```

That should be colons, rather than semicolons.
```suggestion
	"Erreur %s : rien que le désert ici...": "Fehler %s: hier ist nur Wüste...",
```